packages/flutter/test/widgets/heroes_test.dart

```diff
@@ -2399,6 +2399,7 @@ Future<void> main() async {
       ),
     );
     final HeroController controller = HeroController();
+    addTearDown(controller.dispose);
```

Comment:
why doesn't this fail before?

Reply:
It failed, but the bots are not blocking yet. This is proposal in progress: http://flutter.dev/go/leak-tracker-make-bots-blocking